rpm/gratia-probe.spec

```diff
@@ -607,6 +607,9 @@ This is a transitional dummy package for gratia-probe-slurm; it may safely be re
 %files slurm
 
 %changelog
+* Tue Dec 10 2024 Matt Westphall <westphall@wisc.edu.> -  2.8.5-1
+- Log gratia probe failures to aid debugging (SOFTWARE-6037)
+  
```

Comment:
There's also the fix from https://opensciencegrid.atlassian.net/browse/SOFTWARE-5558